securebanking-openbanking-uk-rs-simulator-server/src/test/resources/messages/swagger.properties

```diff
@@ -1,10 +1,10 @@
 # Swagger Specification endpoint properties
-swagger.title=Payment initiation and Account information APIs Specification
-swagger.description=Swagger for Payment initiation and Account information APIs Specification
+swagger.title=Read/Write Open Banking API Specification (UK)
+swagger.description=Swagger for Read/Write Open Banking API Specification (UK) (https://openbankinguk.github.io/read-write-api-site3)
 swagger.license=open-licence
 swagger.license-url=https://www.openbanking.org.uk/open-licence
 swagger.terms-of-service-url=https://backstage.forgerock.com/knowledge/openbanking/article/a45894685
-swagger.version=v3.1.6
 swagger.contact.name=ForgeRock RS Simulator
 swagger.contact.url=https://www.forgerock.com/
-swagger.contact.email=openbanking-support@forgerock.com
+swagger.docket.apis.basePackage=com.forgerock.securebanking.openbanking.uk.rs.api
+swagger.docket.paths.selector.regex=/open-banking/.*
```

Comment:
Regular expression to publish only the requestMappings that match with this expression and avoid publish internal, admin or backoffice APIs.